website/content/api-docs/secret/consul.mdx

```diff
@@ -81,13 +81,15 @@ updated attributes.
 - `token_type` `(string: "client")` - Specifies the type of token to create when
   using this role. Valid values are `"client"` or `"management"`.
 
-- `policy` `(string: <policy or policies>)` – Specifies the base64 encoded ACL policy. The
-  ACL format can be found in the [Consul ACL
-  documentation](https://www.consul.io/docs/internals/acl). This is
-  required unless the `token_type` is `management`.
+- `policy` `(string: <policy>)` – Specifies the base64-encoded ACL policy. This is
+  required unless the `token_type` is `"management"`. [Deprecated as of Consul 1.4 and 
+  removed as of Consul 1.11.](https://www.consul.io/api/acl/legacy)
+
+- `policies` `(list: <policy or policies>)` – The list of policies to assign to the
```

Comment:
Should we make the mention of "Either policies or roles are required for Consul 1.5 and above." in here as well?

Reply:
Sorry for the delay on getting this committed, I agree it should be mentioned here as well. Added it in the last commits. https://github.com/hashicorp/vault/pull/14014/commits/aa1901b0cb5e6f64ad77cd8f807b2b51195ac534